sktree/tree/_classes.py

```diff
@@ -1206,67 +1194,911 @@ def fit(self, X, y, sample_weight=None, check_input=True):
+
+    Attributes
+    ----------
+    feature_importances_ : ndarray of shape (n_features,)
+        The impurity-based feature importances.
+        The higher, the more important the feature.
+        The importance of a feature is computed as the (normalized)
+        total reduction of the criterion brought by that feature.  It is also
+        known as the Gini importance [4]_.
+
+        Warning: impurity-based feature importances can be misleading for
+        high cardinality features (many unique values). See
+        :func:`sklearn.inspection.permutation_importance` as an alternative.
+
+    max_features_ : int
+        The inferred value of max_features.
+
+    n_features_in_ : int
+        Number of features seen during :term:`fit`.
+
+    feature_names_in_ : ndarray of shape (`n_features_in_`,)
+        Names of features seen during :term:`fit`. Defined only when `X`
+        has feature names that are all strings.
+
+    n_outputs_ : int
+        The number of outputs when ``fit`` is performed.
+
+    tree_ : Tree instance
+        The underlying Tree object. Please refer to
+        ``help(sklearn.tree._tree.Tree)`` for
+        attributes of Tree object.
+
+    min_patch_dims_ : array-like
+        The minimum dimensions of a patch.
+
+    max_patch_dims_ : array-like
+        The maximum dimensions of a patch.
+
+    data_dims_ : array-like
+        The presumed dimensions of the un-vectorized feature vector.
+
+    Notes
+    -----
+    Patches are 2D masks that are applied onto the data matrix. Following sklearn
+    API standards, ``X`` is always a ``(n_samples, n_features)`` array even if
+    X is comprised of images, or multivariate-time series. The ``data_width`` and
+    ``data_height`` parameters are used to inform the ``PatchObliqueDecisionTreeRegressor``
+    of the original structure of the data. It is required that
+    ``data_width * data_height = n_features``.
+
+    When users pass in ``X`` to :meth:`fit`, tt is presumed that all vectorization operations
+    are done C-contiguously (i.e. the last axis is contiguous).
+
+    Note that for a patch height and width of size 1, the tree is exactly the same as the
+    decision tree, albeit with less efficiency optimizations. Therefore, it is always
+    recommended to set the range of patch heights and widths based on the structure of your
+    expected input data.
+
+    References
+    ----------
+    .. footbibliography::
+    >>> from sklearn.datasets import load_diabetes
+    >>> from sklearn.model_selection import cross_val_score
+    >>> X, y = load_diabetes(return_X_y=True)
+    >>> from sktree.tree import PatchObliqueDecisionTreeRegressor as RGS
+    >>> regressor = RGS(random_state=0)
+    >>> cross_val_score(regressor, X, y, cv=10)
+    ...                    # doctest: +SKIP
+    ...
+    array([-0.10163671, -0.78786738,  0.01490768,  0.32737289, -0.24816698,
+            0.41881754,  0.0588273 , -1.48722913, -0.07927208, -0.15600762])
```

Comment:
This is incorrect.

It should be
```suggestion
    References
    ----------
    .. footbibliography::
    
    Examples
    ----------
    >>> from sklearn.datasets import load_diabetes
    >>> from sklearn.model_selection import cross_val_score
    >>> X, y = load_diabetes(return_X_y=True)
    >>> from sktree.tree import PatchObliqueDecisionTreeRegressor as RGS
    >>> regressor = RGS(random_state=0)
    >>> cross_val_score(regressor, X, y, cv=10)
    ...                    # doctest: +SKIP
    ...
    array([-0.10163671, -0.78786738,  0.01490768,  0.32737289, -0.24816698,
            0.41881754,  0.0588273 , -1.48722913, -0.07927208, -0.15600762])
```

Reply:
fixed